fetch.bs

```diff
@@ -813,23 +802,28 @@ the empty string,
 "<code>worker</code>", or
 "<code>xslt</code>". Unless stated otherwise it is the empty string.
 
+<p>A <dfn for=export>script-like destination</dfn> is "<code>script</code>",
+"<code>serviceworker</code>", "<code>sharedworker</code>", or "<code>worker</code>".
```

Comment:
> If we made "xslt" part of this, then nosniff would fail.

I see. Hrm. Ok. :)

> And adding "xslt" to MIME type blocking probably doesn't help either as XSLT already requires a correct MIME type?

 I'm not familiar enough with the XSLT code in Chrome, so I'm not actually sure it does. But I agree it should.

So, yeah. I think you're right. Consider the suggestion dropped. :P